scripts/e2e-maestro.sh

```
@@ -28,21 +28,41 @@ esac
+    local lower_word="${word,,}"
+    result+="${lower_word^}"
+  done
+
+  printf '%s' "$result"
+}
+
+SCHEME="$(to_pascal_case "$PACKAGE_NAME")Example"
 
 if [ "$PLATFORM" == "ios" ]; then
-  cd $EXAMPLE_DIR/ios
+  cd "$EXAMPLE_DIR/ios"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)